badges.go

```diff
@@ -145,4 +145,12 @@ var BADGES = []Badge{
 			return c
 		},
 	},
+	{
+		"first-merge",
+		"First merge",
+		"Has at least one Pull Request merged",
+		func(p *Player) int {
+			return len(p.Merged)
```

Comment:
Returning the length would give a badge not only for the first merge, but for all the merges, incrementing the counter for the badge.
I would prefer this badge to be given only once for the first merge. Could you change the return value to be 0 or 1 only ?

Reply:
As you can see [here](https://hacktoberfes-first-merg-7ny3ee.herokuapp.com/)
![image](https://user-images.githubusercontent.com/853903/97800211-c3463480-1c33-11eb-8e1f-830380ccb811.png)

Reply:
Agree